spec/Configuration/DotenvSpec.hs

```diff
@@ -8,6 +8,8 @@ import Test.Hspec
 
 import System.Environment (lookupEnv)
 import Control.Monad (liftM)
+import Data.Maybe (fromMaybe)
+import Data.Functor ((<$>))
```

Comment:
This could be conditionally imported with [this](https://github.com/stackbuilders/dotenv-hs/blob/master/spec/Configuration/DotenvSpec.hs#L13)